tests/decrosstalk/tmp/.gitignore

```diff
@@ -0,0 +1 @@
+# empty file
```

Comment:
Did you meant to include `/tmp` in the repo?

Reply:
I did originally, but it has since been removed.